lib/exabgp/configuration/neighbor/api.py

```diff
@@ -139,7 +139,7 @@ def _empty (cls):
 	def clear (self):
 		self.api = {}
 		self.named = ''
-		pass
+		Section.clear(self)
```

Comment:
Did you find anything wrong that this fixes please ?

Reply:
Yeah, on 'restart', the follow code https://github.com/Exa-Networks/exabgp/blob/2c27112b480132be12a1b3290704f0312769eef9/lib/exabgp/configuration/neighbor/api.py#L147 returns exception/error like

```
19:31:13 | 1342   | configuration | not reloaded, no change found in the configuration
19:31:13 | 1342   | configuration | problem parsing configuration file line 21
19:31:13 | 1342   | configuration | error message: the name "services" already exists in api
```

Reply:
Could you please, create the new build 4.1.3 and upload it into the PyPI? )